Quizzler-iOS13/Controller/QuizViewController.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  QuizViewController.swift
+//  Quizzler-iOS13
+//
+//  Created by Ronald Ngounou on 4/18/24.
+//  Copyright © 2024 The App Brewery. All rights reserved.
+//
+
+import UIKit
+
+class QuizViewController : UIViewController{
```

Comment:
What is the purpose of this ViewController?

Reply:
I didn't want to modify the initial view controller on the previous application so that I could use it as a template by keeping the same logic to implement the QuizViewController.
I am not too sure about how to connect the outlets of the MainStory board to the QuizViewController though. It seems that by default it goes to ViewController. I have to look this part up.

Would you recommend I create a second viewController or should I keep everything on the old one?